server/tournaments/index.ts

```diff
@@ -1695,6 +1697,17 @@ export const commands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
```

Comment:
Possibly seek out the argument after `export` for future-proofing with HTML, Markdown, et al?